### PR TITLE
0.1.0 update

### PR DESCRIPTION
**Pull request by Pandrex247**
0.10 Update, but also renames all of the artefacts into lowercase for Maven Central and renames one of the Kafka modules to be consistent with the other modules.

### CODE REVIEW

File: AmazonSQS/AmazonSQSJCAAPI/pom.xml [hunk 1]
```diff
@@ -3,12 +3,12 @@
...
-        <version>0.1.0-SNAPSHOT</version>
+        <artifactId>cloud-connectors</artifactId>
+        <version>0.1.0</version>
         <relativePath>../../pom.xml</relativePath>
     </parent>
     <groupId>fish.payara.cloud.connectors.amazonsqs</groupId>
-    <artifactId>AmazonSQSJCAAPI</artifactId>
+    <artifactId>amazon-sqs-jca-api</artifactId>
     <packaging>jar</packaging>
    <name>Amazon SQS JCA Adpater API</name>
```
Comment (michaelranaldo) [inline AmazonSQS/AmazonSQSJCAAPI/pom.xml]: Typo on Adapter

File: MQTT/MQTTRAR/pom.xml [hunk 2]
```diff
@@ -2,13 +2,13 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
     <groupId>fish.payara.cloud.connectors.amazonsqs</groupId>
```
Comment (michaelranaldo) [inline MQTT/MQTTRAR/pom.xml]: Should this be MQTT?

File: Kafka/KafkaJCAAPI/target/maven-archiver/pom.properties [hunk 3]
```diff
@@ -0,0 +1,5 @@
+#Generated by Maven
+#Fri May 26 09:42:01 BST 2017
+version=0.1.0
+groupId=fish.payara.cloud.connectors.kafka
+artifactId=kafka-rar-api
```
Comment (michaelranaldo) [inline Kafka/KafkaJCAAPI/target/maven-archiver/pom.properties]: I think this should be jca?

File: Kafka/KafkaJCAAPI/target/maven-status/maven-compiler-plugin/compile/default-compile/inputFiles.lst [hunk 4]
```diff
@@ -0,0 +1,15 @@
+/home/andrew/Git/Cloud-Connectors/Kafka/KafkaJCAAPI/src/main/java/fish/payara/cloud/connectors/kafka/api/KafkaConnectionFactory.java
```
Comment (Cousjava) [inline Kafka/KafkaJCAAPI/target/maven-status/maven-compiler-plugin/compile/default-compile/inputFiles.lst]: Should this be in here? I think you should add /target to .gitignore

**Pull request merged**